plugins/DevicesDetection/Columns/DeviceBrand.php

```diff
@@ -60,4 +60,15 @@ public function onNewVisit(Request $request, Visitor $visitor, $action)
 
         return $parser->getBrand();
     }
+
+    /**
+     * @param Request $request
+     * @param Visitor $visitor
+     * @param Action|null $action
+     * @return mixed
+     */
+    public function onAnyGoalConversion(Request $request, Visitor $visitor, $action)
```

Comment:
True, there will three columns be added to `log_conversion` table. Can also rebase and recreate the pull request on 3.x